graph.c

```diff
@@ -0,0 +1,145 @@
+//idea: https://runestone.academy/ns/books/published/pythonds3/Graphs/AnAdjacencyList.html
+//Implementation of simple directed cycled weighted graph
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+// #include "graph_hash_table.c"
+
+#define GRAPH_VERTICES_TABLE_SIZE 10
+
+typedef struct {
+    struct Graph_vertice_neighbor *next;
+    char *name;
+    int weight;
+} Graph_vertice_neighbor;
```

Comment:
What's the difference between Graph_vertice_neighbor and Graph_vertice? The neighbor has weight which... maybe then Weighted_graph_vertice?

Reply:
I'd like to remain my name.
Graph keeps hash table of Graph_vertices
Graph_vertice keeps name and dynamic array of neighbors

Thus I named it Graph_vertice_neighbor